### PR TITLE
Amp stories

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: src/stories/10_Introduction/_Hello_World.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!-- 
+  ## Introduction
+
+  How to tell stories with AMP.
```
Comment (kul3r4) [inline src/stories/10_Introduction/_Hello_World.html]: is there a link to a doc already?

File: src/stories/10_Introduction/_Hello_World.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        font-size: 2.875em;
+        font-weight: normal;
+        line-height: 1.174;
+        text-transform: uppercase;
+      }
+    </style>
+  </head>
+
+  <body>
+    <!-- ## Writing the Story -->
```
Comment (kul3r4) [inline src/stories/10_Introduction/_Hello_World.html]: Would that make sense to use a simple AMP tag, like `amp-img` just to say that AMP stories support AMP tags as normal AMP pages?

File: src/stories/10_Introduction/_Hello_World.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    <!-- This is the story tag -->
+    <amp-story standalone>
+      <!-- This is the cover page -->
+      <amp-story-page id="cover">
+        <amp-story-grid-layer template="vertical">
```
Comment (kul3r4) [inline src/stories/10_Introduction/_Hello_World.html]: are there more details on the `amp-story-grid-layer` like which other templates it support?

File: templates/css/example.css [hunk 4]
```diff
@@ -95,6 +95,24 @@ code {
 .abe-code-preview {
   background: #f9f7f7;
 }
+.abe-story-preview {
```
Comment (kul3r4) [inline templates/css/example.css]: does it make sense to have css for stories in a separate file and add it to the page only if it's a story instead for all the pages?

File: src/stories/10_Introduction/_Hello_World.html (hunk 2 above)
Comment (sebastianbenz) [reply src/stories/10_Introduction/_Hello_World.html]: They don't amp-story must be the only child of body.

File: src/stories/10_Introduction/_Hello_World.html (hunk 3 above)
Comment (sebastianbenz) [reply src/stories/10_Introduction/_Hello_World.html]: This would go into different samples...

File: templates/css/example.css (hunk 4 above)
Comment (sebastianbenz) [reply templates/css/example.css]: I wouldn't that this is justified as long as it's only 5 extra lines

File: src/stories/10_Introduction/_Hello_World.html (hunk 2 above)
Comment (kul3r4) [reply src/stories/10_Introduction/_Hello_World.html]: ok! thanks for clarifying

File: src/stories/10_Introduction/_Hello_World.html (hunk 3 above)
Comment (kul3r4) [reply src/stories/10_Introduction/_Hello_World.html]: ok

**Comment by sebastianbenz** [comment]
Split this into two PRs (first one is #1156)

**Pull request closed without merging**